### PR TITLE
Scheduler: Construct the model/view architecture

### LINKED ISSUE #182: Scheduler: Construct the model/view architecture
### Feature you want to implement
As the first step of #181, I will construct the model/view architecture in order to show the fixed scheduled queue.
Updating the queue actively won't be under this issue.

#### ScheduleModel: A subclass of `QAbstractTableModel`
It is the main point of this issue.
This model manages the scheduled queue internally and make the view show it appropriately.

Each row is an experiment, and each column is an info of a specific experiment; rid, status, priority, pipeline, due_date, and expid.

### Additional context
The implementation will be based on [PyQt docs](https://doc.qt.io/qtforpython-5/overviews/model-view-programming.html#a-read-only-example-model).

### PR DESCRIPTION
**Pull request by BECATRUE**
This closes #182.

As the first step of a new scheduler, I constructed the basic model/view architecture.
For test, I just set scheduled queue directly in temporary.

### Screenshot
image

### CODE REVIEW

File: iquip/apps/scheduler.py [hunk 1]
```diff
@@ -1,477 +1,121 @@
...
-
-        Fetches the index of the selected element.
-        
-        Args:
-            index: The ModelIndex instance containing 
-              row, column, and etc. values of the selected element.
         
-        Returns:
-            A QMimeData instance that holds str value of the index.
+        DisplayRole: Returns column-th info field of row-th experiment in scheduleList.
```
Comment (kangz12345) [inline iquip/apps/scheduler.py]: Please start the return value section with `"Returns:"` just like the others.

File: iquip/apps/scheduler.py [hunk 2]
```diff
@@ -1,477 +1,121 @@
...
-        super().__init__(parent=parent)
-        self.ip = ip
-        self.port = port
-        self.fetched.connect(callback, type=Qt.QueuedConnection)
-
-    def run(self):
+        if not index.isValid():
+            return QVariant()
+        if role != Qt.DisplayRole:
+            return QVariant()
```
Comment (kangz12345) [inline iquip/apps/scheduler.py]: Will these be updated later? If not, combining them would be better using `or`.

File: iquip/apps/scheduler.py [hunk 3]
```diff
@@ -1,477 +1,121 @@
...
+        return getattr(self.scheduleList[row], infoField)
+
+    def headerData(self, section: int, orientation: Qt.Orientation, role: Qt.ItemDataRole) -> Any:
         """Overridden.
-
-        Fetches the experiment list as a dictionary from the proxy server,
-          and emits a list and an ExperimentInfo instance for display.
+        
+        Horizontal: Returns the corresponding info field.
+        Vertical: Returns an index started from 1.
```
Comment (kangz12345) [inline iquip/apps/scheduler.py]: Please follow the return value docstring style.

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: iquip/apps/scheduler.py (hunk 2 above)
Comment (BECATRUE) [reply iquip/apps/scheduler.py]: It will be updated later, but it may be a far futere. So, I will combine them.

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**